src/Microsoft.Framework.PackageManager/Publish/PublishRoot.cs

```diff
@@ -106,7 +106,7 @@ private void GenerateBatchFiles()
             }
 
             const string template = @"
-@""{0}{1}.exe"" --appbase ""%~dp0{2}"" Microsoft.Framework.ApplicationHost {3} %*
+@""{0}{1}.exe"" --appbase ""%~dp0{2}"" Microsoft.Framework.ApplicationHost --configuration {3} {4} %*
```

Comment:
Do you mind if I merge this first then follow up the change to use string interpolation?